batch/batch/server/server.py

```diff
@@ -81,15 +81,29 @@ def make_logger():
 
 class JobTask:  # pylint: disable=R0903
     @staticmethod
-    def copy_task(job_id, task_name, files):
-        if files:
-            container = kube.client.V1Container(image='alpine',
-                                                name=task_name,
-                                                command=['echo', 'hello'])
-
-            spec = kube.client.V1PodSpec(containers=[container],
-                                         restart_policy='Never')
-
+    def copy_task(job_id, task_name, files, copy_service_account_name):
+        if files is not None:
+            assert copy_service_account_name is not None
+            authenticate = 'gcloud -q auth activate-service-account --key-file=/gcp-sa-key/key.json'
+            copies = ' & '.join([f'gsutil cp {src} {dst}' for (src, dst) in files])
+            wait = 'wait'
```

Comment:
What is the `wait` for?

Reply:
Each of those `gsutil` commands runs in the background (nb the `' & '`), `wait` with no arguments [waits for all child processes to complete](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/wait.html).

Reply:
oh cool! I prob should have known that.